inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasUimaParser.java

```diff
@@ -232,78 +213,162 @@ else if (annotation instanceof Sentence) {
+                                    mtasAnnotationTypeFeatureLabel.addPositionRange(beginToken,
+                                            endToken);
+                                    tokenCollection.add(mtasAnnotationTypeFeatureLabel);
+                                    indexedStr = getIndexedName(annotationUiName) + "."
+                                            + getIndexedName(feature.getUiName())
+                                            + MtasToken.DELIMITER + kbValue;
+                                    
+                                    // Indexing UI annotation without type i.e Concept/Instance
+                                    log.debug("Indexed String without type for : {}", indexedStr);
+                                    MtasToken mtasAnnotationFeatureLabel = new MtasTokenString(
+                                            mtasId++, indexedStr, beginToken);
+                                    mtasAnnotationFeatureLabel.setOffset(annotation.getBegin(),
+                                            annotation.getEnd());
+                                    mtasAnnotationFeatureLabel.addPositionRange(beginToken,
+                                            endToken);
+                                    tokenCollection.add(mtasAnnotationFeatureLabel);
+                                    
+                                    // Indexing UI annotation without type and layer for generic search
+                                    indexedStr = IndexingConstants.kbEntity + MtasToken.DELIMITER + kbValue;
+                                    log.debug("Indexed String without type and label for : {}", indexedStr);
+                                    MtasToken mtasAnnotationKBEntity = new MtasTokenString(
+                                            mtasId++, indexedStr, beginToken);
+                                    mtasAnnotationKBEntity.setOffset(annotation.getBegin(),
+                                            annotation.getEnd());
+                                    mtasAnnotationKBEntity.addPositionRange(beginToken,
+                                            endToken);
+                                    tokenCollection.add(mtasAnnotationKBEntity);
+                                }
                             }
                         }
                     }
                 }
             }
-        }
-        catch (Exception e) {
+        } catch (Exception e) {
             log.error("Unable to index document", e);
         }
         return tokenCollection;
     }
 
-    @Override
-    public String printConfig()
+    /**
+     * Replaces space with underscore n a {@code String}
+     * @param {@code String} uiName
+     * @return {@code String}
+     */
+    public String getIndexedName(String uiName)
     {
+        String indexedName = uiName.replace(" ", "_");
+        return indexedName;
+    }
+
+    @Override
+    public String printConfig() {
+        return null;
```

Comment:
Document why null is returned.

Reply:
Not being used, but I am not sure why exactly it was written. @betoboullosa can we remove this method and `SimpleAnnotationLayer` class in `MtasUimaParser`?

Reply:
Its the method from abstract class `MtasParser'

Reply:
Not used, method from abstract class 'MtasParser'